src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTodos } from '../../context/TodosContext';
+import { TempTodo } from '../TempTodo';
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, filterStatus } = useTodos();
+
+  const filteredTodos = todos.filter(todo => {
```

Comment:
`TodoList` is a representational component, it shouldn't have any logic regarding filtering. It accepts todos via props and renders them, also can render temp todo as it's related to todos